Application/EdFi.Ods.Api/Controllers/DataManagementControllerBase.cs

```diff
@@ -239,6 +248,7 @@ public virtual async Task<IActionResult> Put([FromBody] TPutRequest request, Gui
             }
 
             Response.Headers[HeaderConstants.ETag] = GetEtag(result.ETag).ToString();
+            Response.GetTypedHeaders().ContentType = new MediaTypeHeaderValue(MediaTypeNames.Application.Json);
```

Comment:
Why do we need to set ContentType and decorate with the Produce attribute?  Aren't they doing the same thing?

Reply:
profiles has a differnt content type, and the decoration breaks post and put

Reply:
ohh i see what you saying i will remove this line on put and post since we are decorating the method